packages/app-mobile/components/NoteEditor/CodeMirror/RegionSpec.ts

```diff
@@ -0,0 +1,153 @@
+/**
+ * Describes an inline region's formatting. For example, an italicized region.
+ */
+
```

Comment:
The code in this whole class doesn't look great, is that copied and pasted from elsewhere? Like the C-style declaration of variables (they should instead be declared just before they are needed, and preferable as const), the use of non-strict equality, etc.

Reply:
No. This class was an attempt to factor messy regular-expression-related logic into a unit-testable `struct`.

> C-style declaration of variables (they should instead be declared just before they are needed, and preferable as const), the use of non-strict equality, etc.

My for-school programming has been in C/C++/Java (and I'm still used to `var` keyword scoping)! I'll keep this in mind.